src/api/hooks/video.ts

```diff
@@ -121,32 +121,11 @@ export const useMostViewedVideos = (
   variables?: GetMostViewedVideosQueryVariables,
   opts?: MostViewedVideosQueryOpts
 ) => {
```

Comment:
Cannot comment on it, but there's a `useMostViewedVideosIds` function above that doesn't make sense anymore